src/utils.ts

```diff
@@ -37,15 +37,14 @@ export function parseNumber(s: string): number | undefined {
 }
 
 export function parseParameters(parameterOverrides: string): Parameter[] {
-  try {
-    const path = new URL(parameterOverrides)
-    const rawParameters = fs.readFileSync(path, 'utf-8')
+  if (parameterOverrides.trim().startsWith('file://')) {
+    const path = parameterOverrides.trim().replace('file://', './')
```

Comment:
👆 if you address the behavior you should do a `path.resolve` here, because not all inputs as you have discovered are the same. There are cases where users use `GITHUB_WORKSPACE` to create the path.